### ISSUE TITLE
How to enable the use of the timer hack

### ISSUE THREAD

**Issue by dannsystems** [NoCheatPlus/Issues#653]
#### Short description of the issue.
Basically I use this plugin on my anarchy server to stop game breaking hacks being used but I would like timer to be enabled. I am not sure where or how to do this as there is in timer category anywhere in the config

#### Have you made any (noteworthy) changes to the default configuration of NCP?
No

#### Do you run any other special plugins that affect game mechanics next to NC+ (adding skills, machines, adding/changing blocks, other anti cheating)?
No

#### Provide additional information, such as error/debug/violation logs. (For reproducible issues with moving consider: https://github.com/NoCheatPlus/Docs/wiki/Debugging#on-the-fly-debug-output-for-individual-players) Use GitHub gists or http://pastebin.com/ or similar for huge logs, or attach a (zip-) file here. For image uploads, please prefer a site that allows viewing without JavaScript, you can upload file attachments for issues right here.

Basically I just need to know where I can enable the allowance of the timer/speed hack please.

**Comment by Janmm14** [comment]
BLOCKBREAK_FREQUENCY, BLOCKBREAK_FASTBREAK, BLOCKINTERACT_SPEED, BLOCKPLACE_FASTPLACE, BLOCKPLACE_SPEED, INVENTORY_FASTCONSUME, INVENTORY_INSTANTEAT, MOVING_MOREPACKETS, MOVING_VEHICLE_MOREPACKETS, NET_ATTACKFREQUENCY, NET_FLYINGFREQUENCY, NET_KEEPALIVEFREQUENCY

**Comment by dannsystems** [comment]
thanks
